### ISSUE TITLE
when i use bazel to create aar instead the one in your project, it doesnot work

### ISSUE THREAD

**Issue by athenason** [jiuqiant/mediapipe_multi_hands_tracking_aar_example#3]
I use the following command to create aar, but when i instead it into your project, it crashed when run on my phone, but your original project works well. Could you please tell me the bazel command and build file you create the aar?

the following is mine.

bazel command
bazel build -c opt --host_crosstool_top=@bazel_tools//tools/cpp:toolchain --fat_apk_cpu=arm64-v8a,armeabi-v7a //mediapipe/examples/android/src/java/com/google/mediapipe/apps/aar_example:mp_multi_hand_aar

bazel version 2.0.0
android sdk version 28
ndk version 20.1.5948944
build file
load("//mediapipe/java/com/google/mediapipe:mediapipe_aar.bzl", "mediapipe_aar")

mediapipe_aar(
    name = "mp_multi_hand_aar",
    calculators = ["//mediapipe/graphs/hand_tracking:multi_hand_mobile_calculators"],
)

**Comment by cosimo17** [comment]
Same issus. did you solve this issue?

**Comment by Misby** [comment]
Same issus

**Comment by LegPro** [comment]
Yes, Same issue. it will be great if all mediapipe aar commands gets shared to all.

**Comment by LegPro** [comment]
I followed the steps provided here "https://google.github.io/mediapipe/getting_started/android_archive_library.html" to create error.
I created all things successfully. They mention a graph file
load("//mediapipe/java/com/google/mediapipe:mediapipe_aar.bzl", "mediapipe_aar")

mediapipe_aar(
    name = "mp_face_detection_aar",
    calculators = ["//mediapipe/graphs/face_detection:**_mobile_calculators_**"],
)

Then I went to that location **_//mediapipe/graphs/face_detection_**  , In BUILD file i found cc_library mentioned in mediapipe_arr : **_mobile_calculators_**.


In above view, 

for hand tracking i seen above command :
load("//mediapipe/java/com/google/mediapipe:mediapipe_aar.bzl", "mediapipe_aar")

mediapipe_aar(
name = "mp_multi_hand_aar",
calculators = ["//mediapipe/graphs/hand_tracking:_**multi_hand_mobile_calculators**_"],
)

Then i went to that location //mediapipe/graphs/hand_tracking and tried to find cc_library with name  **_multi_hand_mobile_calculators_** but I didnt find any. Can some one clear my doubt regarding this? how you guys know you have to use **_multi_hand_mobile_calculators_** ? I saw one BUILD in that directory which contain cc_library name ="mobile_calculators". Please help me understand this selection of calculators.

**Comment by hannesa2** [comment]
I re-built the complete chain (Thanks to this repo ❤️) by my own based on mediaPipe `0.8.2`
This pull request https://github.com/hannesa2/mediapipe/pull/31 archives an artifact.
<img width="450" alt="image" src="https://user-images.githubusercontent.com/3314607/110255761-a03ae100-7f95-11eb-9a50-441d8195e9c9.png">

Which I use here https://github.com/hannesa2/mediapipeMultiHandsTracking/releases/tag/0.5

Maybe it helps you, when you have all with comprehensible builds

**Comment by chensisi0730** [comment]
i think connected to  edition ， in  v0.8.8  , it is  "mobile_calculators"
